### PR TITLE
Show missing chapter count between two chapters in chapter list

### LINKED ISSUE #8460: Show missing chapters in list
### Describe your suggested feature

Ideally, in the chapter list of any entry, there should be a visual clue that there are missing chapters

Similiar, but not the same: #6183, #6486, #8446

First got the idea from seeing Antsy post about it:
image

Ghost gave a simple mockup 
image

I made a simple mockup using TF's mogub editor
image

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.14.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by AntsyLich**
Closes #8460

Screenshot_1698778285

### CODE REVIEW

File: app/src/main/java/eu/kanade/presentation/manga/MangaScreen.kt [hunk 1]
```diff
@@ -750,92 +760,122 @@ private fun SharedMangaBottomActionMenu(
...
+                Row(
+                    modifier = Modifier.padding(
+                        horizontal = MaterialTheme.padding.medium,
+                        vertical = MaterialTheme.padding.small,
+                    ),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(MaterialTheme.padding.medium),
+                ) {
+                    HorizontalDivider(modifier = Modifier.weight(1f))
+                    Text(
```
Comment (arkon) [inline app/src/main/java/eu/kanade/presentation/manga/MangaScreen.kt:798]: Maybe we could use the secondary alpha for the text too? It stands out a bit too much right now IMO.

File: app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt [hunk 2]
```diff
@@ -1005,6 +1008,25 @@ class MangaScreenModel(
                 chapters.applyFilters(manga).toList()
             }
 
+            val chapterListItems by lazy {
+                processedChapters.insertSeparators { chapterItem, chapterItem2 ->
+                    if (chapterItem2 == null) return@insertSeparators null
+
+                    when {
+                        chapterItem == null -> floor(chapterItem2.chapter.chapterNumber).toInt().minus(1).coerceAtLeast(0)
+                        manga.sortDescending() -> calculateChapterGap(chapterItem.chapter, chapterItem2.chapter)
```
Comment (arkon) [inline app/src/main/java/eu/kanade/tachiyomi/ui/manga/MangaScreenModel.kt:1017]: I'm not sure if this is quite correct, it shows things as missing at the top of the list.

File: app/src/main/java/eu/kanade/presentation/manga/MangaScreen.kt (hunk 1 above)
Comment (AntsyLich) [reply app/src/main/java/eu/kanade/presentation/manga/MangaScreen.kt:798]: Better?
Screenshot_1698821487

Comment (arkon) [reply app/src/main/java/eu/kanade/presentation/manga/MangaScreen.kt:798]: LGTM

**Comment by AntsyLich** [comment]
Changes pushed. Ready to merge.

**Pull request merged**